### DISCUSSION TITLE
Suggest moving default forum to github.com/orgs/ringecosystem/discussions

### DISCUSSION THREAD

**Discussion by hackfisher** [darwinia-network/darwinia#1593] [General]
Only use this for archive purpose and Darwinia chain specific topics (including KtonDAO before it setup its own forum).

https://github.com/orgs/ringecosystem/discussions

**Comment by hackfisher** [comment]
Related: https://github.com/orgs/ringecosystem/discussions/60

**Reply by SasoLithops** [reply]
I get page not found when clicking the link. I think mistake is link should be from other organisation not ringecosystem.

**Comment by boundless-forest** [comment]
I completely agree with this. Perhaps we can transfer those valuable discussions there.

**Comment by aurexav** [comment]
Also need to update https://github.com/darwinia-network/DIPs

**Reply by aurexav** [reply]
Or transfer DIPs to ringdao
